### ISSUE TITLE
Feature request: Add data from power meter

### ISSUE THREAD

**Issue by kfhdk** [time4tea/gopro-dashboard-overlay#70]
It would be nice to have overlay data from a power meter

Example of a single entry in pgx-file from strava:
```
   <trkpt lat="55.2359580" lon="10.4869030">
    <ele>78.8</ele>
    <time>2022-07-09T07:09:31Z</time>
    <extensions>
     <power>103</power>
     <gpxtpx:TrackPointExtension>
      <gpxtpx:atemp>21</gpxtpx:atemp>
      <gpxtpx:hr>113</gpxtpx:hr>
      <gpxtpx:cad>96</gpxtpx:cad>
     </gpxtpx:TrackPointExtension>
    </extensions>
   </trkpt>
```
Thanks for this wonderful tool

**Comment by time4tea** [comment]
This is a great suggestion. Thanks for the example file. I dont have a power meter, but I do know there are different types. I'll see if I can find any examples of left/right power meters as well.

**Comment by kfhdk** [comment]
https://github.com/time4tea/gopro-dashboard-overlay/compare/main...kfhdk:gopro-dashboard-overlay:power

**Comment by time4tea** [comment]
This should now be available in 0.65.0
